### DISCUSSION TITLE
EmpiricalFormula returns error for operand - but not +

### DISCUSSION THREAD

**Discussion by samcphee** [OpenMS/OpenMS#6143] [Code]
Hi,

I am trying to take water away from the chemical formula of a peptide which is cyclized however it returns an error.  Below is a simple example which returns the same error.

Is this a known issue or is there a workaround?

```
EmpiricalFormula(str("H2O1") + str("O1"))
>>>H2O2
EmpiricalFormula(str("H2O1") - str("O1"))
>>>TypeError: unsupported operand type(s) for -: 'str' and 'str'
```

**Comment by jpfeuffer** [comment]
Hi! This is indeed a known issue that was spotted a few weeks ago. #5914
It should be fixed in the current nightly wheels on GitHub and will come fixed in  pyopenms 3.0 in the future.

**Reply by samcphee** [reply]
Thanks!
